javascript-javascript1-week3.js

```diff
@@ -0,0 +1,181 @@
+// Its warmup time!
+// Item array removal
+
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+];
+const nameToRemove = names.splice(1, 1);
```

Comment:
This works, but it's not very dynamic in case you want to make the code more useful for the future. For example if you suddenly need to remove "Samuel" or another name, you would have to find it's index. In that context, another solution would be to use an array method called `indexOf()`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf

Reply:
Thanks for the tips